explore/src/clue/scala/queries/common/ProgramUserWithRoleSubquery.scala

```diff
@@ -14,7 +14,12 @@ object ProgramUserWithRoleSubquery
   override val subquery: String = s"""
     {
       user $ProgramUserSubquery
-      partner
+      partner {
```

Comment:
What type in the schema is this a query for?  `ProgramUser`?  Shouldn't it be `programLink:` here?

Reply:
Yes, how did this work 🤔

Reply:
Fixed